### PR TITLE
Homedir

### PR DESCRIPTION
**Pull request by Mycelin**
Simplicity: 14 bytes less.

### CODE REVIEW

File: preptrack/preptrack [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-SAVE_TO="/home/$(whoami)/Music"
+SAVE_TO="~/Music"
```
Comment (bacondropped) [inline preptrack/preptrack]: ~ is not expanded within double quotes, this will make a ./~/Music directory.

**Comment by Mycelin** [comment]
Rly? Try eval echo "~"

**Comment by bacondropped** [comment]
`eval` evaluates its input "one more time" as opposed to direct variable assignment. I mean, why don't you just test the patch? I did, and it produced exactly that result.

**Comment by Mycelin** [comment]
Use $HOME then

**Comment by bacondropped** [comment]
Already did!

**Pull request closed without merging**